services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers_extension.py

```diff
@@ -281,14 +281,10 @@ async def attach_container_to_network(
             )
             return
 
-        try:
-            network = await docker.networks.get(item.network_id)
-        except aiodocker.docker.DockerError as e:
-            raise HTTPException(
-                status.HTTP_404_NOT_FOUND,
-                detail="No network with id={item.network_id} found",
-            ) from e
-
+        # NOTE: Not querying for the network because it might not be on the host.
+        # Attaching it to a container will make the object appera on the host.
+        # This issue presents itself when dealing with worker nodes.
```

Comment:
What about:
```A docker network is only visible on a docker node when it is used by a container```

And one question, you are not handling the case when the network really does not exist. is this ok? before you were catching the exceptions